hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -97,7 +98,8 @@
                // ENUMASSIGN issues can cause LEC warnings later down the road (see #10083
                // and #10952 for context). The warning is therefore promoted to an error
                // in simulations in order to catch this as early as possible.
-               "-error=ENUMASSIGN"
+               "-error=ENUMASSIGN",
+               "{vcs_debug_access_build_opts}"
```

Comment:
Super minor nit: could you move this above, before the error promotion options (rationale is to have all error promotions appear together at the end of VCS invocation).

Reply:
Done, thanks